rosdep/base.yaml

```diff
@@ -3232,6 +3252,12 @@ libpqxx-dev:
     xenial: [libpqxx-dev]
     yakkety: [libpqxx-dev]
     zesty: [libpqxx-dev]
+libprocps-dev:
+  debian: [libprocps-dev]
+  ubuntu: [libprocps-dev]
+libprocps6:
+  debian: [libprocps6]
+  ubuntu: [libprocps6]
```

Comment:
libprocps6 is only available in Debian Stretch and Ubuntu Bionic respectively. Other versions of this library appear to be used on earlier and later platforms. You'll need to at least scope this key to the distributions where it is available. I'm not sure if we should consider a `libprocps` key that maps to the available version for each distribution.

Reply:
You are right, we shouldn't make it available for each distribution since they literally doesn't exist in other versions. I made fix such that make libprocps6 only available to stretch and bionic respectively in 5bc928b.

Reply:
Updated comment https://github.com/ros/rosdistro/pull/24685#issuecomment-622704138 accordingly.